src/Compilers/Lua/Portable/Parser/Lexer.cs

```diff
@@ -65,7 +66,7 @@ public override void Dispose()
             base.Dispose();
         }
 
-        public SyntaxToken Lex() => QuickScanSyntaxToken() ?? LexSyntaxToken();
+        public SyntaxToken Lex() => LexSyntaxToken();
```

Comment:
Instead of deleting, add `#if LEXER_TOKEN_CACHE` so that it can be easily re-enabled without having to revert this commit as this is a temporary hotfix.